R/Mexico.R

```diff
@@ -21,8 +21,12 @@ Mexico <- R6::R6Class("Mexico",
     # Core Attributes
     #' @field level_1_region the level 1 region name
     level_1_region = "estados",
-    #' @field level_2_region the level 2 region name.
+    #' @field level_2_region the level 2 region name
```

Comment:
not to make additional changes but as @RichardMN suggests in #217 perhaps these should both also be lists? I am very open to either option?

Reply:
As in `region_levels = list("level_1_region" = "something", "level_2_regions" = "something_else")` ?

Reply:
agreed, a list would be fine, will do

Reply:
shall we have a vote I have no idea.

Reply:
it wil be harder to document and program if we define it as a list but maybe easier longer term?

Reply:
i'd think documentation is okay since it's the same principle across the list so still makes sense. trying out a list now

Reply:
list implemented in latest commit. works fine and looks a bit neater and more integrated IMO (nice to have an clear name linked with the "localise" generic class field) - so keen on that, thanks :)